src/EPR.Calculator.API/Validators/CreateDefaultParameterDataValidator.cs

```diff
@@ -5,7 +5,8 @@
 
 namespace api.Validators
```

Comment:
Have to update the namespace formatting to match with other EPR projects. Something to consider in the future.

Reply:
Done